source/backend/dal/Repositories/CompensationRequisitionRepository.cs

```diff
@@ -53,6 +53,7 @@ public PimsCompensationRequisition GetById(long compensationRequisitionId)
                 .Include(x => x.AcquisitionOwner)
                 .Include(x => x.AcquisitionFilePerson)
                 .Include(x => x.InterestHolder)
+                .Include(x => x.AlternateProject)
```

Comment:
this might not be necessary given that the front end can query the project by id.

Reply:
It's necessary in order to display the Alternate Project information in the View for the comp req.